cmd/src/campaigns_common.go

```diff
@@ -68,6 +68,8 @@ func newCampaignsApplyFlags(flagSet *flag.FlagSet, cacheDir string) *campaignsAp
 		&caf.namespace, "namespace", "",
 		"The user of organization namespace to place the campaign within.",
 	)
+	flagSet.StringVar(&caf.namespace, "n", "", "Alias for -namespace.")
```

Comment:
does that mean `-n` takes precendence over `-namespace`? Just curious, not a blocker

Reply:
I don't think so (or I'm misunderstanding what you mean). They're independent flags and if you give them different pointers, they'll work independently.

Reply:
I mean we're copying into `&caf.namespace` for both flags 

So if someone would provide `-n eseliger -namespace=mrnugget`, would namespace be `eseliger` or `mrnugget` is what I meant. I suppose it is `eseliger`?

Reply:
Last one wins:

<img width="987" alt="screenshot_2020-08-28_14 38 07@2x" src="https://user-images.githubusercontent.com/1185253/91561548-1ce06900-e93c-11ea-8716-24d774099561.png">

Reply:
Interesting 👍 thanks